exercises/concept/card-tricks/.docs/introduction.md

```diff
@@ -3,11 +3,29 @@
 Slices in Go are similar to lists or arrays in other languages. They hold a number of elements of a specific type (or interface).
 
 Slices in Go are based on arrays. Arrays have a fixed size. A slice, on the other hand, is a dynamically-sized, flexible view into the elements of an array.
-Technically a slice consists of a pointer to a start element in an array and a length. This makes for example creating a sub-slice `s[3:7]` very efficient.
-But don't worry: most of the time this knowledge is not needed in order to work with slices.
 
 A slice is written like `[]T` with `T` being the type of the elements in the slice:
 
 ```go
-var s []int
+var empty []int                 // an empty slice
+withData := []int{0,1,2,3,4,5}  // a slice pre-filled with some data
 ```
+
+You can get/set an element at a given zero-based index using square-bracket notation:
+
+```go
+withData[1] = 5
+x := withData[1] // x is now 5
+```
+
+You can create a new slice from an existing slice by getting a range of elements,
+once again using square-bracket notation, but specifying both a starting (inclusive) and ending (exclusive) index.
```

Comment:
nit: Prefer to keep the sentences together on the same line otherwise we end up having a larger blast radius when reviewing changes because you have to change additional lines that were unaffected by the actual change. The next two lines are a good example of how this works best.

Reply:
done